src/test/java/at/ac/tuwien/kr/alpha/test/util/TestUtils.java

```diff
@@ -95,4 +117,75 @@ public static Atom basicAtomWithSymbolicTerms(String predicate, String... consta
 		return new BasicAtom(pred, trms);
 	}
 
+	private static SystemConfig regressionTestConfigToSystemConfig(RegressionTestConfig cfg) {
+		SystemConfig retVal = new SystemConfig();
+		retVal.setGrounderName(cfg.getGrounderName());
+		retVal.setSolverName(cfg.getSolverName());
+		retVal.setNogoodStoreName(cfg.getNoGoodStoreName());
+		retVal.setSeed(cfg.getSeed());
+		retVal.setBranchingHeuristic(cfg.getBranchingHeuristic());
+		retVal.setDebugInternalChecks(cfg.isDebugChecks());
+		retVal.setEvaluateStratifiedPart(cfg.isEvaluateStratifiedPart());
+		retVal.setGrounderAccumulatorEnabled(cfg.isDisableInstanceRemoval());
+		retVal.setUseNormalizationGrid(!cfg.isEncodeAggregatesUsingSortingGrid());
```

Comment:
Maybe a topic for another discussion (especially since the aggregate rewriting is not yet merged): why is there the boolean negation `!` needed? When tracing this setting, I saw three different names used for this (counting grid, sorting grid, normalization grid) and sometimes the one was the other but inverted. This should be cleaned up (but maybe not here, but in the one for the aggregate rewriting).
I just would like to be sure that the negation here is correct.